TurkishFilter/sections/css_extended.txt

```diff
@@ -9,6 +9,7 @@ vizyonfilmizle.tv#$?#.fancybox-overlay-fixed[-ext-has="> div"] { display: none!i
 vizyonfilmizle.tv#$?#.fancybox-lock body[-ext-has="> div"] { overflow: visible!important; }
 vizyonfilmizle.tv#$?#html[-ext-has="> body"] { overflow: visible!important; }
 !
+nurmutfagi.de#?#.sb-widget > .widget-title:contains(Reklam):upward(1)
```

Comment:
better something like that  `nurmutfagi.de#?#.sb-right > div.sb-widget:has(> div.ad-widget)`